Sources/MapboxNavigation/CameraController.swift

```diff
@@ -56,22 +56,6 @@ class CameraController: NavigationComponent, NavigationComponentDelegate {
         navigationMapView.navigationCamera.moveToOverview()
     }
     
-    @objc func recenter(_ sender: AnyObject) {
```

Comment:
I think `CameraController` was originally implemented to separate concerns, and looks like it'd be better to keep re-centering logic here. Can you please try to use something like:
```swift
navigationViewData.containerViewController as? NavigationViewController)?.navigationComponents
```
to access all existing components from `CameraController`?

Reply:
For example:
```swift
let navigationViewController = navigationViewData.containerViewController as? NavigationViewController
navigationViewController?.navigationComponents.compactMap({ $0 as? NavigationMapInteractionObserver }).forEach {
    $0.navigationViewController(didCenterOn: location)
}
```

Reply:
Yes. I moved it back to the `CameraController` .